### DISCUSSION TITLE
Logical operator or in complex media query

### DISCUSSION THREAD

**Discussion by dabidesh** [mdn/mdn-community#272] [Content] [answered]
Can it be more clearly written that the logical operators for complex media query are 4 – not, and, only and or(,)(not only comma)?

Yes, there is only one example with or:
https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#testing_for_multiple_features_with_or
But it doesn't and isn't mentioned here that both work – or and comma:
https://developer.mozilla.org/en-US/docs/Web/CSS/@media#logical_operators

Or and comma work and are valid css.

**Comment by teoli2003** [comment, marked as answer]
Hi, @dabidesh.

Thanks for the request. Can you open an issue for this (under mdn/content)?

**Reply by dabidesh** [reply]
Hi, @teoli2003.
I forgot to mention that I opened an issue: https://github.com/mdn/content/issues/21959
